packages/core/src/providers/query/index.test.tsx

```
@@ -3,10 +3,26 @@ import { describe, it, expect, vi } from 'vitest';
+}));
+
+vi.mock('../../universal/tracker', () => ({
+  useTracker: () => ({
+    captureError: vi.fn(),
+  }),
+}));
 
-// Mock the useSubscription hook
 vi.mock('../universal/hooks/useSubscription', () => ({
-  useSubscription: () => ({
+  useSubscription: ({}: SubscriptionParams) => ({
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)